Logger/SerilogConfiguration.cs

```diff
@@ -0,0 +1,29 @@
+using Serilog;
+
+namespace QuickEdit.Logger;
+
+public class SerilogConfiguration
+{
+
+	public static void ConfigureLogger()
+	{
+		var intermediateOutputPath =
+			Path.GetFullPath(Path.Combine(AppDomain.CurrentDomain.BaseDirectory, @"..\..\..", @"obj\Debug\net8.0\Logs"));
+
+		Directory.CreateDirectory(intermediateOutputPath);
+
+		var logPath = Path.Combine(intermediateOutputPath, $"consoleapplog-{DateTime.UtcNow:yyyyMMddHHmmss}.txt");
```

Comment:
The log is saved in a location only appropriate in Visual Studio. On a server, it would create the logs in a hard to find path, and might cause confusion.

I'd recommend changing it to just `logs`, as it should be clear to the user where the logs are located.
That would also eliminate the need for the `intermediateOutputPath` variable, as just "logs" could be used, though a better approach, I think, is still using a variable, but renaming it.

Another thing I've noticed, is the log names contain the date twice, and are not really specifying what program they're from. From my testing, it seems that Serilog (or something else idk) automatically adds the date to the filename, removing the need for manually adding the date and time.

```suggestion
		var logDirectory = "logs";

		Directory.CreateDirectory(logDirectory);

		var logPath = Path.Combine(logDirectory, $"quickedit-.log");
```